adm/Cargo.toml

```diff
@@ -29,9 +29,11 @@ clap = ">=2.29.0"
 libc = ">=0.2.35"
 lmdb-zero = ">=0.4.1"
 protobuf = "2.0"
+sawtooth = { version = "0.6", features = ["client-rest"] }
```

Comment:
This should be an optional dependency, for now, and it should be included under an experimental feature below.  Here, it can just say optional, where as when it's use, the experimental feature (possibly just called `client-cli`) can use it as such:

```
client-cli = ["sawtooth/client-rest", ...]
```

Reply:
Fixed